azure-pipelines.yml

```diff
@@ -103,29 +127,53 @@ jobs:
 
   - script: |
       export LLVM_CONFIG_PATH=$(brew --prefix llvm)/bin/llvm-config
-      ./scripts/run_ci_checks.sh fmt
+      export RUSTFLAGS="-D warnings"
+      export RUSTDOCFLAGS="-D warnings"
+      cargo fmt --check
     displayName: 'cargo fmt --check'
   
   - script: |
```

Comment:
I would skip this step on Darwin since we're mostly interested in detecting broken builds. That would also cut down on duplication. If you insist on keeping it, I would refer to the comment for cargo check above rather than duplicating the comment.

Reply:
See https://github.com/immunant/c2rust/pull/696#issuecomment-1272655223.

Reply:
> Re-duplicating checks on Darwin and Linux, I thought about it a bit more carefully. cargo fmt takes about ~2s and cargo doc about ~10s, so there's not too much benefit in removing those.

The benefit is in making this script shorter. As I've said, we are mostly interested in catching broken builds and tests. Remember that when reviewer and author disagree, break in favor of reviewer.

Reply:
Okay, I'll remove it.

Reply:
Correctly rebased now.